### PR TITLE
ran clang-format on the entire codebase

### PR DESCRIPTION
**Pull request by yarneo**

### CODE REVIEW

File: components/ActivityIndicator/src/private/MaterialActivityIndicatorStrings_table.h [hunk 1]
```diff
@@ -9,10 +9,10 @@
...
 
 static NSString *const kMaterialActivityIndicatorStringTable[] = {
-  @"MaterialActivityIndicatorInProgressAccessibilityValue",   // In Progress
-  @"MaterialActivityIndicatorAccessibilityLabel",   // Activity Indicator
-  @"MaterialActivityIndicatorProgressHaltedAccessibilityValue",   // Progress Halted
-  @"MaterialActivityIndicatorProgressCompletedAccessibilityValue",   // %ld Percent Complete
+    @"MaterialActivityIndicatorInProgressAccessibilityValue",         // In Progress
+    @"MaterialActivityIndicatorAccessibilityLabel",                   // Activity Indicator
+    @"MaterialActivityIndicatorProgressHaltedAccessibilityValue",     // Progress Halted
+    @"MaterialActivityIndicatorProgressCompletedAccessibilityValue",  // %ld Percent Complete
```
Comment (jverkoey) [inline components/ActivityIndicator/src/private/MaterialActivityIndicatorStrings_table.h:15]: This is 🔥

**Review by jverkoey — APPROVED** [review]
LGTM with the caveat that we cut a hotfix release with only this change.

File: components/ActivityIndicator/src/private/MaterialActivityIndicatorStrings_table.h (hunk 1 above)
Comment (yarneo) [reply components/ActivityIndicator/src/private/MaterialActivityIndicatorStrings_table.h:15]: yaaa! 😍

**Comment by jverkoey** [comment]
Detected lots of bugs in the apidiff toolchain as well!

**Comment by jverkoey** [comment]
Filed https://github.com/material-motion/apidiff/issues/27 to track.

**Review by randallli — APPROVED** [review]

**Pull request merged**